pagopa/io-app
# fix: [IOBP-962] Remove native header into payment method details error

## Short description
This PR fixes the header while showing an error inside the payment method details error screen.

## List of changes proposed in this pull request
- Disabled the `headerShown` property to this screen;
- Removed the hook that sets undefined the header since it's not needed anymore;

## How to test
- Open the Wallet section;
- Disable your wifi or stop the dev-server;
- Tap on a payment method to navigate in its details page;
- Check that the native header is not showed anymore;


## Preview
|Before|After|
|-|-|
|<img src="https://github.com/user-attachments/assets/0cf6c36f-b5a4-45f4-9eeb-3f69eb3531b8" width="350" />|<img src="https://github.com/user-attachments/assets/e5deed9e-a648-4c93-bdbf-54fc8fc6b78d" width="350" />
